docs/guide/distributed-tracing.asciidoc

```diff
@@ -0,0 +1,14 @@
+[[distributed-tracing]]
+== Distributed tracing
+
+Elastic APM supports distributed tracing to enable you to analyze performance through your microservices architecture by tracing all the requests from the initial web request to your front-end service,
+to queries made to your back-end services all in one view.
+This makes finding possible bottlenecks through-out your application much easier and faster.
+There's no additional configuration needed for distributed tracing,
+but please ensure you are using the latest version of the agent.
+
+The APM UI in Kibana also supports displaying traces and an updated Timeline visualization has been redesigned to show all the transactions from individual services that are connected in a trace.
```

Comment:
comma after "displaying traces"?

Reply:
This sentence was probably too long as well. Broke it up